draft-ietf-scitt-architecture.md

```diff
@@ -593,15 +591,17 @@ Hence, the Registry may contain both Transparent Statements and governance entri
 
 ## Verifying Transparent Statements {#validation}
 
-For a given Artifact, Verifiers take as trusted inputs:
+For a given Transparent Statement, Verifiers take as trusted inputs:
 
 1. the distributed identifier of the Issuer (or its resolved key manifest),
-1. the expected name of the Artifact (i.e., the Feed),
+1. the collection of Transparent Statements to which this Statement about the Artifact belongs (i.e., the Feed),
```

Comment:
Don't see this working?  The verification of a transparent statement should never require binding to a feed.  I thought we were talking about feeds for auditors not end users.  If so, making it that end user needs to see the Transparency Service directly and "subscribe" has issues.